lib/repl.js

```diff
@@ -96,7 +96,7 @@ class LineParser {
   }
 
   parseLine(line) {
-    var previous = null;
+    let previous = null;
```

Comment:
We're still generally using `var` over `let` for performance reasons. I'm not sure how much that should matter in the REPL though.